rotkehlchen/tests/api/test_user_assets.py

```diff
@@ -952,13 +958,17 @@ def test_exporting_user_assets_list(rotkehlchen_api_server, globaldb, with_custo
 @pytest.mark.parametrize('start_with_logged_in_user', [True])
 @pytest.mark.parametrize('method', ['post', 'put'])
 @pytest.mark.parametrize('file_type', ['zip', 'json'])
-def test_importing_user_assets_list(rotkehlchen_api_server, method, file_type):
+def test_importing_user_assets_list(
+        rotkehlchen_api_server: APIServer,
+        method: str,
+        file_type: str,
+):
     """Test that the endpoint for importing user assets works correctly"""
     dir_path = Path(__file__).resolve().parent.parent
     if file_type == 'zip':
-        filepath = dir_path / 'data' / 'exported_assets.zip'
+        filepath = os.path.join(dir_path, 'data', 'exported_assets.zip')
```

Comment:
This should not be needed. Why exactly is it faiing here? `dir_path` is a `Path` and as such should be OS agnostic: https://docs.python.org/3/library/pathlib.html

Something else must be wrong on this one.

What's more, `os.path.join` already returns a string and you stringify it again later down (or well the previous code did).

But generally we need to figure out why this is wrong here. It should work.

Reply:
You're right. This one actually passes. I think this is a remnant from my early iterations where changing this (somehow) fixed the test, but it passes now without this change so I'll change it back.